pages.pt_PT/common/batch.md

```diff
@@ -0,0 +1,16 @@
+# batch
+
+> Executar comandos num momento mais tarde quando a carga do sistema permitir.
+> O serviço atd (ou atrun) deve correr para atuais execuções.
+
+- Executar comandos do standard input (premir `Ctrl + D` quando terminado):
+
+`batch`
+
+- Executar um comando do standard input:
+
+`echo "{{./make_db_backup.sh}}" | batch`
```

Comment:
Since we are translating the "caminho/para/ficheiro" perhaps we should be translating all parameter names. I'll ask you to change this to something in Portuguese.

Reply:
@schneiderl Thanks for the input, I've corrected that bit!
@sbrl sorry about that, will do next time!

Reply:
@schneiderl anything else that I need to do for this one?